docs/src/meshes/MeshDataMeshes.md

```diff
@@ -0,0 +1,70 @@
+* geometric quantities in `MeshData` are stored as matrices of dimension ``\text{number of points per element } \times \text{number of elements}``. 
+
+Quantities in `rd::RefElemData`: 
+* `rd.Np, rd.Nq, rd.Nf`: the number of nodal interpolation points, volume quadrature points, and face quadrature points on the reference element, respectively. 
+* `rd.Vq`: interpolation matrices from values at nodal interpolation points to volume quadrature points
+* `rd.wq`: volume quadrature weights on the reference element
+* `rd.Vf`: interpolation matrices from values at nodal interpolation points to face quadrature points
+* `rd.wf`: a vector containing face quadrature weights on the reference element
+* `rd.M`: the quadrature-based mass matrix, computed via `rd.Vq' * diagm(rd.wq) * rd.Vq`.
+* `rd.Pq`: a quadrature-based ``L^2`` projection matrix `rd.Pq = rd.M \ rd.Vq' * diagm(rd.wq)` which maps between values at quadrature points and values at nodal points. 
+* `Dr, Ds, Dt` matrices are nodal differentiation matrices with respect to the ``r,s,t`` coordinates, e.g., `Dr*f.(r,s)` approximates the derivative of ``f(r,s)`` at nodal points. 
+
+Quantities in `md::MeshData`: 
+* `md.xyz` is a tuple of matrices `md.x`, `md.y`, `md.z`, where column `e` contains coordinates of physical interpolation points. 
+* `md.xyzq` is a tuple of matrices `md.xq`, `md.yq`, `md.zq`, where column `e` contains coordinates of physical quadrature points. 
+* `md.rxJ, md.sxJ, ...` are matrices where column `e` contains values of ``J\frac{\partial r}{\partial x}``, ``J\frac{\partial s}{\partial x}``, etc. at nodal interpolation points on the element `e`.
+* `md.J` is a matrix where column `e` contains values of ``J`` at nodal interpolation points.
+* `md.sJ` is a matrix where column `e` contains values of the face Jacobian (e.g., determinant of the geometric mapping between a physical face and a reference face) at face quadrature points.
+* `md.nxJ, md.nyJ, ...` are matrices where column `e` contains values of components of the unit normal scaled by the face Jacobian `md.sJ` at face quadrature points.
+
+For more details, please see the [StartUpDG.jl docs](https://jlchan.github.io/StartUpDG.jl/dev/). 
+## Special options
+
+Trixi solvers on simplicial meshes use `DG` solver types with the basis field set as a `RefElemData` type. By default, `RefElemData(Tri(), N)` will generate data for a degree ``N`` polynomial approximation on a triangle. There are also several parameters which can be tweaked:
+
+* `RefElemData(Tri(), N, quad_rule_vol = quad_nodes(Tri(), Nq))` will substitute in a volume quadrature rule of degree `Nq` instead of the default (which is a quadrature rule of degree `N`). Here, a degree `Nq` rule will be exact for at least degree `2*Nq` integrands (such that the mass matrix is integrated exactly).
```

Comment:
Can you specify a quadrature that is exact for polynomials of odd degree (such as Lobatto-Legendre quadrature)? Would that make sense at all?

Reply:
Yes, 'quad_nodes' just calls another quadrature routine with argument '2*Nq'. I can make a note of that in the docs. 

I defaulted to 'quad_nodes' returning a quadrature rule of degree '2*Nq' to make the behavior in 1d ('quad_nodes(N)' returns an N+1 Gauss Legendre rule exact for at least a degree 2N integrand) consistent with 2D.

Reply:
Sounds good 👍An additional note would be perfect.